### PR TITLE
Add support for node js 8+

### PR DESCRIPTION
**Pull request by daniellmorris**
Add node js 8+ support

### CODE REVIEW

File: doc/vebugger.txt [hunk 1]
```diff
@@ -300,6 +302,21 @@ directory. This will display a list of available processes to attach to.
 Notice that unlike GDB, you need "src" here since Mdbg doesn't display full
 source file paths.
 
+LAUNCHING NINSPECT                                             *vebugger-ninspect*
+
+RDebug is launched with *vebugger#ninspect#start*
```
Comment (idanarye) [inline doc/vebugger.txt]: `RDebug` -> `NInspect`

File: doc/tags [hunk 2]
```diff
@@ -0,0 +1,68 @@
+:VBGclearBreakpints	vebugger.txt	/*:VBGclearBreakpints*
```
Comment (idanarye) [inline doc/tags]: Please don't commit tag files

Comment (daniellmorris) [reply doc/tags]: Fixed. I removed tag.s file.

File: doc/vebugger.txt (hunk 1 above)
Comment (daniellmorris) [reply doc/vebugger.txt]: OK, I fixed that.

**Pull request merged**